lib/raw_dynamic.dart

```diff
@@ -60,7 +60,7 @@ void _handleOnBeginFrame(Duration timeStamp) {
       );
     final Picture picture = recorder.endRecording();
 
-    final SceneBuilder sceneBuilder = SceneBuilder()
+    final SceneBuilder sceneBuilder = SceneBuilder(view.viewId)
```

Comment:
Overall this is great!

For my own understanding: Is the SceneBuilder taking a view ID just an artifact of this being a prototype? Or is this the plan for the final implementation as well? There were some concerns about tying a Scene to a view in https://github.com/flutter/flutter/issues/112202#issuecomment-1362250535.

Reply:
You're right, this is still up for discussion.